### PR TITLE
refact: vhs github action

### PR DESCRIPTION
**Pull request by jeevithakannan2**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Documentation update
- [x] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
- Refactor to use vhs github action 

## Testing
- Tested on linutil fork.

## Impact
- The installation of the fonts take too much time. A workaround must be done for the fonts.

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by adamperkowski** [comment]
Wow this takes waaaay longer than the previous one.

**Review by adamperkowski — APPROVED** [review]

**Comment by adamperkowski** [comment]
It installs a lot of unused fonts

**Comment by nnyyxxxx** [comment]
i like how the preview is navigating to the utilities tab when you can just search any option in any tab

**Comment by nnyyxxxx** [comment]
isnt system clean up in system setup

**Comment by adamperkowski** [comment]
> i like how the preview is navigating to the utilities tab when you can just search any option in any tab

image

File: .github/workflows/linutil.yml [hunk 1]
```diff
@@ -97,7 +97,6 @@ jobs:
         uses: charmbracelet/vhs-action@v2.1.0
         with:
           path: "docs/assets/preview.tape"
-          install-fonts: "true"
```
[the remark this answers is not in the document]
Comment (jeevithakannan2) [reply .github/workflows/linutil.yml]: It will automatically install both the versions

[the remark this answers is not in the document]
Comment (adamperkowski) [reply .github/workflows/linutil.yml]: If you say so. Hope it's tested.

[the remark this answers is not in the document]
Comment (jeevithakannan2) [reply .github/workflows/linutil.yml]: Yep always tested :grin:

**Review by nnyyxxxx — APPROVED** [review]

**Comment by jeevithakannan2** [comment]
> Wow this takes waaaay longer than the previous one. https://github.com/jeevithakannan2/linutil/actions/runs/11156719955 Yeah the previous one was 1 minute 40-something seconds This one is 4 minutes

Now it takes only around 2min just like before.

**Comment by adamperkowski** [comment]
> Now it takes only around 2min just like before.

Thanks :))

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**